mc-conflicts-ui/src/setupTests.ts

```diff
@@ -8,3 +8,5 @@ import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
+global.URL.createObjectURL = jest.fn();
+HTMLCanvasElement.prototype.getContext = jest.fn();
```

Comment:
Please add a comment why do we mock the following functions

Reply:
fixed in ea0d181